package.json

```
@@ -0,0 +1,5 @@
+{
+  "devDependencies": {
+    "prettier": "^2.8.4"
```

```suggestion
    "prettier": "^2.7.1"
```
Apparently, the pre-commit mirror is a bit behind - but probably it's preferable to have the same versions in CircleCI and in the pre-commit hook?